### ISSUE TITLE
Django 1.9 Warnings

### ISSUE THREAD

**Issue by Nekmo** [jezdez/django-hosts#32]
Most of my console warnings are triggered by Django-hosts:

    /home/nekmo/.virtualenvs/exiscontacts/src/django-hosts/django_hosts/resolvers.py:14: RemovedInDjango19Warning: django.utils.importlib will be removed in Django 1.9.
      from django.utils.importlib import import_module

    /home/nekmo/.virtualenvs/exiscontacts/src/django-hosts/django_hosts/resolvers.py:31: RemovedInDjango19Warning: memoize wrapper is deprecated and will be removed in Dj
    ango 1.9. Use django.utils.lru_cache instead.
      get_hostconf = memoize(get_hostconf, _hostconf_cache, 0)

    /home/nekmo/.virtualenvs/exiscontacts/src/django-hosts/django_hosts/resolvers.py:38: RemovedInDjango19Warning: memoize wrapper is deprecated and will be removed in Dj
    ango 1.9. Use django.utils.lru_cache instead.
      get_hostconf_module = memoize(get_hostconf_module, _hostconf_module_cache, 1)

    /home/nekmo/.virtualenvs/exiscontacts/src/django-hosts/django_hosts/resolvers.py:51: RemovedInDjango19Warning: memoize wrapper is deprecated and will be removed in Dj
    ango 1.9. Use django.utils.lru_cache instead.
      get_host = memoize(get_host, _host_cache, 1)

    /home/nekmo/.virtualenvs/exiscontacts/src/django-hosts/django_hosts/resolvers.py:62: RemovedInDjango19Warning: memoize wrapper is deprecated and will be removed in Dj
    ango 1.9. Use django.utils.lru_cache instead.
      get_host_patterns = memoize(get_host_patterns, _host_patterns_cache, 0)

**Comment by AliHichem** [comment]
same here.

**Comment by timgraham** [comment]
Addressed in https://github.com/jezdez/django-hosts/pull/28.

**Comment by iqmaker** [comment]
Warnings have now ( django==1.8.2,  and git+git://github.com/jezdez/django-hosts.git version 1.2 ), how to fix it?

**Comment by int-ua** [comment]
```
django_hosts/resolvers.py:14: RemovedInDjango19Warning: django.utils.importlib will be removed in Django 1.9.
  from django.utils.importlib import import_module
```
is not yet fixed.

**Comment by timgraham** [comment]
It's fixed in #35.

**Comment by aalebedev** [comment]
Hi! How about a new release (without these warnings)?
